pkg/util/validation/rule.go

```diff
@@ -1,4 +1,4 @@
-package interpret
+package validation
```

Comment:
Personally, it's not appropriate to put this file in the current directory, because it's not mainly for validation.

Reply:
Which directory should this file be placed in?

Reply:
/cc @ikaven1024

Reply:
How about `pkg/configurableinterpreter`. Besides this logic is similar, Also can move here (You can do it i another PR):https://github.com/karmada-io/karmada/blob/b1977be9e58f16d132359482739913540c2da808/pkg/resourceinterpreter/configurableinterpreter/configmanager/accessor.go#L70